webpack.config.js

```diff
@@ -14,14 +14,20 @@ module.exports = {
 		root: 'React',
 		commonjs2: 'react',
 		commonjs: 'react',
-		amd: 'react',    	
+		amd: 'react',
     },
     'react-dom': {
 		root: 'ReactDOM',
 		commonjs2: 'react-dom',
 		commonjs: 'react-dom',
 		amd: 'react-dom'
-  	}
+    },
+    'prop-types': {
+      commonjs: 'prop-types',
+      commonjs2: 'prop-types',
+      amd: 'prop-types',
+      root: 'PropTypes',
+    }
```

Comment:
if these are all externals - do they appear in package.json as `peerDependencies` ? 

From memory this "externals" section in the config strips them out from the dist bundle, so we should have these in `peerDependencies`... Looking at the package.json file - they don't seem to be there. I think we should add them in.

Reply:
Yep, that's pretty much what webpack externals do. Good point, I'll add them in. The reason I didn't was because React itself is not set as a peerDependency, but that's not a good reason.